bundle/jsky.app.ot/src/main/resources/resources/conf/UIInfo.xml

```diff
@@ -387,15 +387,24 @@
     <!--
       uiClassName="jsky.app.ot.gemini.flamingos2.EdCompInstFlamingos2"/>
     -->
-    <!--
+
+    <UIInfo
```

Comment:
Is this still used?

Reply:
No. There are also a lot of classes of the form `*GUI` that are completely unused (replaced with `*Form` instead), but I didn't want to mess too much with the unrelated code.

I don't mind cleaning it up in a separate PR once we get this test release out.